meilisearch-release-changelog/v1.10.0.md

```diff
@@ -0,0 +1,374 @@
+
+Done by @ManyTheFish in #4819.
+
+### Index settings
+
+Use the newly introduced `localizedAttributes` setting (here is an example of handling multi-language documents):
+
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "localizedAttributes": [
+      {"locales": ["jpn"], "attributePatterns": ["*_ja"]},
+      {"locales": ["eng"], "attributePatterns": ["*_en"]},
+      {"locales": ["cmn"], "attributePatterns": ["*_zh"]},
+      {"locales": ["fra", "ita"], "attributePatterns": ["latin.*"]},
+      {"locales": [], "attributePatterns": ["*"]}
+    ]
+}'
+```
+
+`locales` is a list of language codes to assign to a pattern, the supported codes are: `epo`, `eng`, `rus`, `cmn`, `spa`, `por`, `ita`, `ben`, `fra`, `deu`, `ukr`, `kat`, `ara`, `hin`, `jpn`, `heb`, `yid`, `pol`, `amh`, `jav`, `kor`, `nob`, `dan`, `swe`, `fin`, `tur`, `nld`, `hun`, `ces`, `ell`, `bul`, `bel`, `mar`, `kan`, `ron`, `slv`, `hrv`, `srp`, `mkd`, `lit`, `lav`, `est`, `tam`, `vie`, `urd`, `tha`, `guj`, `uzb`, `pan`, `aze`, `ind`, `tel`, `pes`, `mal`, `ori`, `mya`, `nep`, `sin`, `khm`, `tuk`, `aka`, `zul`, `sna`, `afr`, `lat`, `slk`, `cat`, `tgl`, `hye`.
+
+`attributePattern` is a pattern that can start or end with a `*` to match one or several attributes.
+
+`"locales": [], "attributePatterns": ["*"]` means the is the default rule.
+
+Notes:
+- if an attribute matches several rules, only the first rule in the list will be applied
+- if the locales list is empty, then Meilisearch is allowed to auto-detect any language in the matching attributes
+- These rules are applied to the `searchableAttributes`, the `filterableAttributes`, and the `sortableAttributes`.
+
+### At search time
+
+The search route accepts a new parameter, `locales` allowing the end-user to define the language used in the current query:
+
+```bash
+curl \
+ -X POST http://localhost:7700/indexes/movies/search \
+  -H 'Content-Type: application/json' \
+ --data-binary '{
+  "q": "進撃の巨人",
+  "locales": ["jpn"]
+}'
+```
+
+The `locales` parameter overrides eventual `locales` in the index settings.
+
+## Experimental: edit documents by using a function
+
+You can edit documents by executing a [Rhai function](https://rhai.rs/) on all the documents of your database or a subset of them that you can select by a Meilisearch filter.
+
+Activate the experimental feature:
+
+```bash
+curl \
+ -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json' \
+ --data-binary '{
+    "editDocumentsByFunction": true
+ }'
+```
+
+By indexing this [movies dataset](https://www.meilisearch.com/movies.json), you can run the following Rhai function on all of them. This function will uppercase the titles of the movies with an `id` > 3000 and add sparkles around it. Rhai templating syntax is applied here:
```

Comment:
Then query the `/documents/edit` route with the editing function:

Reply:
I'm not sure I understand, what should I change?

Reply:
Duh, I forgot to make it an actual suggestion, sorry. Fixed in the original comment.